app.js

```diff
@@ -0,0 +1,179 @@
+  },
+
+  // simulates the number of cookies bought in each hour of a work day
+  simCookiesHrs: function() {
+    this.cookiesHrs = [];
+    var cookiesBought;
+    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
+      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
+      this.cookiesHrs.push(cookiesBought);
+    }
+  },
+};
+
+var centerStore = {
+  location: 'Seattle Center',
+  openAt: 6, // 6 am
+  closeAt: 20, // 8 pm
+  minCustHr: 11,
+  maxCustHr: 38,
+  avgCookieCust: 3.7,
+  cookiesHrs: [],
+
+  // calculates random number of customers for one hour
+  randomCustHr: function() {
+    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
+  },
+
+  // simulates the number of cookies bought in each hour of a work day
+  simCookiesHrs: function() {
+    this.cookiesHrs = [];
+    var cookiesBought;
+    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
+      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
+      this.cookiesHrs.push(cookiesBought);
+    }
+  },
+};
+
+var capitolStore = {
+  location: 'Capitol Hill',
+  openAt: 6, // 6 am
+  closeAt: 20, // 8 pm
+  minCustHr: 20,
+  maxCustHr: 38,
+  avgCookieCust: 2.3,
+  cookiesHrs: [],
+
+  // calculates random number of customers for one hour
+  randomCustHr: function() {
+    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
+  },
+
+  // simulates the number of cookies bought in each hour of a work day
+  simCookiesHrs: function() {
+    this.cookiesHrs = [];
+    var cookiesBought;
+    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
+      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
+      this.cookiesHrs.push(cookiesBought);
+    }
+  },
+};
+
+var alkiStore = {
+  location: 'Alki',
+  openAt: 6, // 6 am
+  closeAt: 20, // 8 pm
+  minCustHr: 2,
+  maxCustHr: 16,
+  avgCookieCust: 4.6,
+  cookiesHrs: [],
+
+  // calculates random number of customers for one hour
+  randomCustHr: function() {
+    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
+  },
+
+  // simulates the number of cookies bought in each hour of a work day
+  simCookiesHrs: function() {
+    this.cookiesHrs = [];
+    var cookiesBought;
+    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
+      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
+      this.cookiesHrs.push(cookiesBought);
+    }
+  },
+};
+
+/** turns the number of hours into an am/pm time **/
+function numToTime(num) {
+  if (num == 0) {
+    return '12am';
+  } else if (num < 12) {
+    return num + 'am';
+  } else if (num == 12) {
+    return '12pm';
+  } else {
+    return (num - 12) + 'pm';
+  }
+}
+
+/** prints an unordered list of the hourly sale simulations to the page **/
+function printStoreSalesSimsList(store) {
+  var position = document.getElementById('store-data');
+
+  if (position) {
```

Comment:
Since you hard code the div with a class of store-data into your html, this will always be true.  Since this is always true, you don't need the if statement at all, just run the code inside.